index.js

```diff
@@ -42,6 +44,10 @@ Toolkit.run(
           tools.log('PR title ' + title + ' does not contain approved project with format [PROJECT-1234]')
           return false
         }
+        if (title.match(BLACKLISTED_TITLE_REGEX)) {
```

Comment:
how abou failing if `head_branch` (replacing '/' with spaces) is detected in the title? it could be a ninja branch that doesnt start with `hotfix` nor `feature`, (some people use `bug/xxxx`as well)

Reply:
Seems a good idea. Removing underscores and dashes per spaces.